### PR TITLE
9068 standalone location filter

### PR DESCRIPTION
**Pull request by ujahio**
Screen Shot 2021-11-02 at 12 54 07 PMScreen Shot 2021-11-02 at 12 54 38 PM

### CODE REVIEW

File: web-client/integration-tests/journey/docketClerkViewsTrialSessionsTab.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { formattedTrialSessions as formattedTrialSessionsComputed } from '../../src/presenter/computeds/formattedTrialSessions';
 import { runCompute } from 'cerebral/test';
 import { trialSessionsHelper as trialSessionsHelperComputed } from '../../src/presenter/computeds/trialSessionsHelper';
+
```
Comment (rachelschneiderman) [inline web-client/integration-tests/journey/docketClerkViewsTrialSessionsTab.js]: Nit: can this be reverted?

File: web-client/src/presenter/computeds/trialCitiesHelper.js [hunk 2]
```diff
@@ -11,30 +10,54 @@ import { state } from 'cerebral';
  */
 export const trialCitiesHelper = (get, applicationContext) => procedureType => {
   const { TRIAL_CITIES } = applicationContext.getConstants();
+  const standaloneRemote = 'Standalone Remote';
```
Comment (rachelschneiderman) [inline web-client/src/presenter/computeds/trialCitiesHelper.js]: I believe there is an entity constant for this value, can this be updated to use that?

File: web-client/src/views/StartCaseInternal/CaseInformation.jsx [hunk 3]
```diff
@@ -131,6 +131,7 @@ export const CaseInformation = connect(
             </label>
           </div>
         </FormGroup>
+        procedure type {form.procedureType}
```
Comment (rachelschneiderman) [inline web-client/src/views/StartCaseInternal/CaseInformation.jsx]: Nit: does this need to be left in?

Comment (TomElliottFlexion) [reply web-client/src/views/StartCaseInternal/CaseInformation.jsx]: Nope good catch, thank you!

File: web-client/src/views/StartCase/TrialCity.jsx [hunk 4]
```diff
@@ -64,18 +64,34 @@ export const TrialCity = connect(
           onChange={onChange}
         >
           {showDefaultOption && <option value="">-- Select --</option>}
-          {Object.keys(trialCitiesByState).map(stateName => (
-            <optgroup key={`TrialCity-${stateName}`} label={stateName}>
-              {trialCitiesByState[stateName].map(trialCity => (
+          {trialCitiesByState.map(locationGroup => {
```
Comment (rachelschneiderman) [inline web-client/src/views/StartCase/TrialCity.jsx]: One suggestion, to reduce complexity in the jsx and in `trialCitiesHelper`; the helper could export a `includeStandalone` flag and then the jsx could be responsible for appending that option on to the beginning of the select list if that flag was true.

Comment (TomElliottFlexion) [reply web-client/src/views/StartCase/TrialCity.jsx]: Refactored! Does this make more sense?

**Review by rachelschneiderman — APPROVED** [review]

File: web-client/src/views/StartCase/TrialCity.jsx (hunk 4 above)
Comment (rachelschneiderman) [reply web-client/src/views/StartCase/TrialCity.jsx]: I believe so! Thanks!

**Pull request merged**